python_modules/libraries/dagster-databricks/dagster_databricks/databricks_pyspark_step_launcher.py

```diff
@@ -45,7 +50,7 @@
         'staging_prefix': Field(
             StringSource,
             is_required=False,
-            default_value='dagster_staging',
+            default_value='/dagster_staging',
```

Comment:
Not a huge deal, but because these changes are independent, it would be preferable to submit them as two separate PRs.

Reply:
I'll create a new PR for this change and the log message fix in the other commit :+1: